backend/src/bundles/users/user.controller.ts

```diff
@@ -60,6 +69,27 @@ class UserController extends BaseController {
             payload: await this.userService.findAll(),
         };
     }
+    /**
+     * @swagger
+     * /users/current:
```

Comment:
Maybe, we should change the route to `/users/:id`? I think it is more common.

Reply:
If we go that way, `getCurrent` handler should become `findById` or `getById`

Reply:
I don’t think that’s correct, because then we’re just taking user by id and not current user. Current user should be taken from the token that client sends to backend

Reply:
I already noticed that there is not need for payload in this case because we are sending token as a header